src/AlgebraicGeometry/Schemes/AffineSchemes/Objects/Attributes.jl

```diff
@@ -480,6 +480,12 @@ julia> codim(Y)
   return dim(ideal(ambient_coordinate_ring(X), [zero(ambient_coordinate_ring(X))])) - dim(X)
 end
 
+function degree(X::Spec{BRT, RT}; check::Bool=true) where {BRT<:Field, RT}
+  @check dim(X) == 0 "The affine scheme X needs to be zero-dimensional"
```

Comment:
```suggestion
  @check dim(X) == 0 "the affine scheme X needs to be zero-dimensional"
```
Really minor thing, but: It's Oscar convention to not use capital letters in the beginning of error messages.

Reply:
(It is?! I didn't know that.)

Reply:
I think, I once learned this from @thofma . But things might have changed in the meantime. Either way, I am reluctant to have the tests run once more only for this. But it's something to keep in mind.